packages/ToggleSwitch/ToggleSwitch.jsx

```diff
@@ -118,12 +133,20 @@ ToggleSwitch.propTypes = {
   /** A callback function to be invoked when the ToggleSwitch button is clicked on.
    @param {SyntheticEvent} event The React `SyntheticEvent` */
   onClick: PropTypes.func.isRequired,
+
+  /** Boolean to automatically focus on ToggleSwitch after interacting with it */
+  autofocus: PropTypes.bool,
```

Comment:
I recommend we remove this prop since it's less code to focus with Refs than to disable and enable this (see second documented example vs the first documented example in ToggleSwitch.md).

Reply:
makes perfect sense!